scripts/fetch-fullnode-ids.js

```diff
@@ -0,0 +1,131 @@
+// Copyright 2025 Hathor Labs
```

Comment:
You can replace this entire file with a simple bash script using `wscat` and `jq`:

```
timeout 5s npx wscat -c ws://fullnode:8080/v1a/event_ws -x '{"type": "START_STREAM", "window_size": 1}' | head -1 | jq -r '"FULLNODE_PEER_ID=" + .peer_id, "STREAM_ID=" + .stream_id' > .identifiers.env
```